persistence/src/main/scala/com/mesosphere/usi/persistence/InMemoryRepositoryImpl.scala

```diff
@@ -0,0 +1,46 @@
+package com.mesosphere.usi.persistence
```

Comment:
For the sake of resolving the related JIRA, I'd go with the core modules only, and add persistence as a follow-up and its own JIRA. There's this fancy persistence impl from @zen-dog in the Marathon project and I wonder whether we should add any dummy impl here now, if we can as well move that one over, with benchmarks and all …

@zen-dog @timcharper thoughts?